components/brave_new_tab_ui/stories/default/data/todayStorybookState.ts

```diff
@@ -0,0 +1,957 @@
+import { boolean } from '@storybook/addon-knobs/react'
+import { BraveTodayState } from '../../../reducers/today'
+
+export default function getTodayState (): BraveTodayState {
+  return {
+    isFetching: boolean('Today is fetching?', false),
+    isUpdateAvailable: boolean('Is Today update available?', false),
+    currentPageIndex: 10,
+    publishers: {
+      ['5eece347713f329f156cd0204cf9b12629f1dc8f4ea3c1b67984cfbfd66cdca5']: {
+        publisher_id: `5eece347713f329f156cd0204cf9b12629f1dc8f4ea3c1b67984cfbfd66cdca5`,
+        publisher_name: `Test Publisher 1`,
+        category: `Tech`,
+        enabled: false,
+        user_enabled: true
+      },
+      ['4eece347713f329f156cd0204cf9b12629f1dc8f4ea3c1b67984cfbfd66cdca5']: {
+        publisher_id: `4eece347713f329f156cd0204cf9b12629f1dc8f4ea3c1b67984cfbfd66cdca5`,
+        publisher_name: `Test Publisher 2`,
+        category: `Top News`,
+        enabled: false,
+        user_enabled: null
+      }
+    },
+    feed: {
+      hash: `123abc`,
+      featuredArticle: {
+        category: `Top News`,
+        publish_time: `2020-10-22 00:09:00 UTC`,
+        url: `https://www.npr.org/2020/10/21/926445682/u-s-blames-iran-for-threatening-election-emails-says-russia-may-interfere-too?utm_medium=RSS&utm_campaign=news`,
+        img: `https://pcdn.brave.software/brave-today/cache/9bbfe86e499867a3988d7b089daa757cb7762a6a9d9f3b9836c4f5b8d8cd610b.jpg.pad`,
+        title: `U.S. Blames Iran For Threatening Election Emails, Says Russia May Interfere Too`,
+        description: `The director of national intelligence and the FBI director said on Wednesday night that U.S. officials believe Iranian influence-mongers are behind an election-intimidation scam.`,
+        content_type: `article`,
+        publisher_id: `5eece347713f329f156cd0204cf9b12629f1dc8f4ea3c1b67984cfbfd66cdca5`,
+        publisher_name: `NPR`,
+        url_hash: `7d36ae2500e2952a49205ee549508046f38aa379b33c138eaa25076447899cd4`,
+        padded_img: `https://pcdn.brave.software/brave-today/cache/9bbfe86e499867a3988d7b089daa757cb7762a6a9d9f3b9836c4f5b8d8cd610b.jpg.pad`,
+        score: 18.525096955122685,
+        points: 18.525096955122685,
+        relative_time: `about 4 hours ago`
+      },
+      featuredDeals: [
```

Comment:
Two small suggestions to make sure the client parsing code is robust:
- Include an article with `padded_img: ""` since that's a valid case when an image cannot be found.
- Include an article without the `padded_img` field at all, in case the backend stops emitting empty fields in the future.